### PR TITLE
UI: remove count api call on every patch in entities pages

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- I worked on removing count API call on every patch action in entities pages

https://github.com/open-metadata/OpenMetadata/assets/66266464/8288ebbd-0cb6-4893-aee9-4f59eaa402b0

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 1]
```diff
@@ -736,7 +738,7 @@ const PipelineDetails = ({
 
   useEffect(() => {
     getEntityFeedCount();
-  }, [pipelineFQN, description, pipelineDetails]);
+  }, [pipelineFQN]);
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:741]: as we are not using pipelineFQN in useEffect, we can remove it. let me know if i am missing something

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:741]: we are getting the `pipelineFQN` by props, and we are expecting it to call the useEffect on pipelineFQN change.

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by Sachin-chaurasiya** [comment]
@Ashish8689 , Can you take care of the cypress failures? thanks

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  done!

**Pull request merged**